### PR TITLE
Fix RichTextEditor story

### PR DESCRIPTION
**Pull request by gjvoosten**
Closes NCI-Agency/anet#3139

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -187,7 +186,7 @@
     "test-jest": "export ANET_CONFIG=../anet.yml ; NODE_ENV=test jest --testPathPattern='tests/(actions|reducers|models)'",
     "test-wdio-ie": "export SERVER_URL=${SERVER_URL:-'http://localhost:8180'} ; NODE_ENV=test node ./tests/util/wdio.ie.js",
     "build-storybook": "build-storybook",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "export STORYBOOK_ANET_CONFIG=/anet.yml ; start-storybook -s .. -p 6006",
```
Comment (cemalettin-simsoft) [inline client/package.json:189]: In other places, we export anet config like this `CONFIG=../anet.yml`, why is this different?

**Review by cemalettin-simsoft — CHANGES_REQUESTED** [review]
The text inside RichText storybook had some inconsistency. Like this, h1 is smaller than h2-h3 etc.
RichText-headers

File: client/package.json (hunk 1 above)
Comment (gjvoosten) [reply client/package.json:189]: It's an HTTP request from the Storybook server (can't access a file directly, as it all runs in the browser). So we just map the top-level ANET directory as Storybook's server's root for serving static files, and request `/anet.yml` from that static area.

**Comment by gjvoosten** [comment]
@cemalettin-simsoft :
> The text inside RichText storybook had some inconsistency. Like this, h1 is smaller than h2-h3 etc.

Looks like Storybook adds global styling like this:
```
h2 {
  font-weight: 900;
  font-size: 32px;
  line-height: 1;
  margin: 0 0 4px;
  display: inline-block;
  vertical-align: top;
}
```
so it's very hard to avoid that problem.

**Review by cemalettin-simsoft — APPROVED** [review]
I thought the problem might come from RichTextEditor component's styling. If the issue is only presented in storybook, then it is probably fine.

**Comment by gjvoosten** [comment]
> I thought the problem might come from RichTextEditor component's styling. If the issue is only presented in storybook, then it is probably fine.

Indeed. It *did* however prompt me to add the missing bootstrap stylesheet that we also include in our global `index.js`. Does not help in this case, but better to be complete.

**Pull request merged**